js/src/lib/interfaces/LibrariesToTasks.ts

```diff
@@ -98,6 +98,25 @@ export const LIBRARY_TASK_MAPPING_EXCLUDING_TRANSFORMERS: Partial<Record<ModelLi
 	"stanza": [
 		"token-classification",
 	],
+	"transformers.js": [
```

Comment:
This is really for the Inference API to display the widgets and buttons for inference. I would remove this part for now

FYI this file's content is automatically created from  https://github.com/huggingface/api-inference-community/pull/158

Reply:
[Updated](https://github.com/huggingface/hub-docs/pull/854/commits/c736384ab1ca1599dbc4dac31f28105d123ce6e7) 👍